script.js

```diff
@@ -0,0 +1,115 @@
+import { filmsSet } from "./data/test-data.js";
+
+const header = {
+  css: "header",
+  paddingX: 20,
+  height: 50,
+  cols: [
+    {
+      view: "label",
+      label: "My App",
+      gravity: 0.2,
+      minWidth: 200
+    },
+    {},
+    {
+      view: "button",
+      type: "icon",
+      icon: "mdi mdi-account",
+      label: "Profile",
+      gravity: 0.2,
+      minWidth: 200
+    }
+  ]
+};
+
+const form = {
+  view: "form",
+  minWidth: 150,
+  maxWidth: 250,
+  elements: [
+    { template: "EDIT FILMS", type: "section" },
+    {
+      margin: 20,
+      cols: [{ view: "label", label: "Title", width: 70 }, { view: "text" }]
```

Comment:
bad practice. you nrrf to use a Text component and set the label property for it

Reply:
understood,
fixed it.

Reply:
not really - you don't need to wrap the text field into the `cols` layout as it is the only element in it

Reply:
Did it